torch/csrc/distributed/c10d/ProcessGroup.cpp

```diff
@@ -186,4 +186,11 @@ void ProcessGroup::init() {
   C10_LOG_API_USAGE_ONCE(fmt::format("c10d.process_group_{}", getBackendName()));
 }
 
+TORCH_LIBRARY(c10d, m) {
+  m.class_<ProcessGroup>("ProcessGroup")
```

Comment:
If you're still getting import errors you might need to move it into the same file as the operator registration. You require this type registration to run before the operator can be registered. But there's no guaranteed order of execution for C++ static initialization from different modules.

Reply:
Good catch.

Reply:
It solved the issue. Thanks, Dima!